packages/discussions/lib/Notifications.js

```diff
@@ -40,7 +40,7 @@ const submitComment = async (comment, discussion, context) => {
     context
   )
 
-  const notifyUsers = await pgdb.query(`
+  const notifyUsers = testUsers || await pgdb.query(`
```

Comment:
`testUsers` may become truthy but not an array and I think it would be sane to check if `testUsers` is an array or not.